cmd/lncli/commands.go

```diff
@@ -111,13 +111,45 @@ var SendCoinsCommand = cli.Command{
 }
 
 func sendCoins(ctx *cli.Context) error {
+	var addr string = ""
+	var amt int64 = 0
+	var args cli.Args = ctx.Args()
+
+	if ctx.NArg() == 0 && ctx.NumFlags() == 0 {
+		cli.ShowCommandHelp(ctx, "sendcoins")
+		return nil
+	}
+
+	if ctx.IsSet("addr") {
```

Comment:
Minor style nit: this if/elseif/else fragment can be re-written as a target-less switch-statement.